tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,112 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached mappable.
+
+"""
+
+###############################################################################
+# Customized Colorbars
+# ====================
+#
+# `matplotlib.colorbar.ColorbarBase` derives from `ScalarMappable` and puts a
+# colorbar in specified axes, it is the base class with standalone colorbar
```

Comment:
I think the original is clearer:
 > ColorbarBase derives from ScalarMappable and puts a colorbar in a specified axes, so it has everything needed for a standalone colorbar.